templates/20-group-validation-webhook.deployment.yaml.tmpl

```diff
@@ -0,0 +1,59 @@
+apiVersion: extensions/v1beta1
```

Comment:
Same comment from the service object, since it will be a single deployment for all the webhooks

Reply:
Agreed 👍